packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -10,21 +10,21 @@
     "vscode": "^1.15.0"
   },
   "dependencies": {
-    "@salesforce/salesforcedx-utils-vscode": "40.11.0"
+    "@salesforce/salesforcedx-utils-vscode": "40.11.0",
+    "request-light": "0.2.1",
```

Comment:
Hoisting these up to the top-level will help with packaging albeit at the cost of some duplicated packages.

The crux is that A -> B -> C. This transitive dependency doesn't bring in all the dependencies when we are using lerna (since it's locally linked).

We can avoid this in the future if we publish everything to the public npm - that would resolve this issue but that would mean that we would need to publish publicly which puts another burden on our ops.